services/users-api/lambdas/deleteAttachment.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable no-console */
+import to from 'await-to-js';
+import { throwError, BadRequestError } from '@helsingborg-stad/npm-api-error-handling';
+import S3 from '../../../libs/S3';
+import * as response from '../../../libs/response';
+import { decodeToken } from '../../../libs/token';
+
+const BUCKET_NAME = process.env.BUCKET_NAME;
```

Comment:
We do not use process.env anywhere else in the code base, should this not be ssmParams?

Reply:
![Skärmavbild 2021-01-28 kl  17 26 54](https://user-images.githubusercontent.com/768077/106167955-0ad65100-618e-11eb-8db4-6ee40b5af3c3.png)
Ser att detta mönster används här så därav mitt val. Tror det var du som myntat idé med att lägga bucket namnet i serverless.yml filen istället för att manuellt lägga in den i SSM.

Tror detta är mer rätt?